students/17th/team1/ChiohKim/user/views.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from django.views     import View
+from django.http      import HttpResponse, JsonResponse
+from django.db.models import Q
+
+from .models      import Account
+
+class SignupView(View):
+    def post(self, request):
+        signup_data = json.loads(request.body)
+
+        try:
+            if Account.objects.filter(email=signup_data['email']).exists() or Account.objects.filter(phonenumber=signup_data['phonenumber']).exists() or Account.objects.filter(name=signup_data['name']).exists():                  
+
+                return JsonResponse({"message":"EXIST_USER"}, status=400) #에러메세지에도 컨벤션이있나요?
```

Comment:
```suggestion
                return JsonResponse({"message":"EXIST_USER"}, status=400) 
```
#에러메세지에도 컨벤션이있나요? 
네 있습니다. 사실 응답은 status code  면 충분합니다. 
하지만 우리는 메세지를 제공하죠 이는 보다 분별력있는 에러 상태 파악을 제공하도록 하는것입니다.
그렇다면 이메세지는 정말 메세지 일까요? 아닙니다. 메세지로 된 코드입니다.
그렇기 때문에 긴 설명이 아닌 단축된 문자코드로서 메세지를 제공하고 수신하는 측(즉, 프론트)과 협의 해야합니다.
아래의 메세지들도 변경 부탁드립니다.

Reply:
넵 수정했습니다!